src/main/java/frc/robot/Robot.java

```diff
@@ -219,10 +281,12 @@ public void robotPeriodic() {
      */
     @Override
     public void autonomousInit() {
-        navX.resetDisplacement();
-        navX.reset();
-
-        ArrayDeque<AutonAction> route = autonRouteChooser.getSelected();
+        AutonRoutes.setupCorrectAutonPaths();
+        ArrayDeque<AutonAction> route = new ArrayDeque<>(autonRouteChooser.getSelected());
+        double delayAmount = SmartDashboard.getNumber("Auton Delay (sec)", 0.0);
+        if (delayAmount > 0) {
+            route.addFirst(new AutonWait(delayAmount));
+        }
```

Comment:
Adds a way for drivers to specify a wait period before auton starts in the SmartDashboard.